rdt-open-reader/src/test/java/io/ona/rdt/robolectric/util/UtilsTest.java

```diff
@@ -223,4 +225,27 @@ public void testIsValidJSONObjectShouldReturnCorrectStatus() {
         Assert.assertFalse(Utils.isValidJSONObject(null));
         Assert.assertFalse(Utils.isValidJSONObject(""));
     }
+
+    @Test
+    public void testVerifyUserAuthorizationShouldExecuteAuthorizationTask() {
+
+        Utils.UserAuthorizationVerificationTask userAuthorizationVerificationTask = Mockito.mock(Utils.UserAuthorizationVerificationTask.class);
```

Comment:
Make this a spy instead of mock to fix coverage.

Reply:
I have written a separate test case because i also need to test the exception case.

Reply:
fixed.